lib/platform/github/index.spec.ts

```diff
@@ -175,6 +175,29 @@ describe('platform/github/index', () => {
       expect(repos).toMatchSnapshot();
       expect(httpMock.getTrace()).toMatchSnapshot();
     });
+    it('should return an array of repos when using Github App endpoint', async () => {
+      //Use Github App token
+      hostRules.find.mockReturnValue({
+        token: 'x-access-token:123test',
+      });
+      httpMock
+        .scope(githubApiHost)
+        .get('/installation/repositories?per_page=100')
+        .reply(200, {
+          repositories: [
+            {
+              full_name: 'a/b',
+            },
+            {
+              full_name: 'c/d',
+            },
+          ],
+        });
+
+      const repos = await github.getRepos();
+      expect(repos).toMatchSnapshot();
```

Comment:
```suggestion
      expect(repos).toStricEqual([
  "a/b",
  "c/d",
]);
```

needs prettify and snapshot update